cli/src/test-utils/test-runner.ts

```diff
@@ -13,472 +13,472 @@ import {
+          `State teardown request failed: ${
+            error.message
+          } (${TestTimer.formattedDiff(testTeardownStartTime)})`,
+          { indent: 2 }
+        );
+      }
+      return false;
+    }
+  }
+
+  /**
+   * Check if an axios response status matches the expected status of a test.
+   *
+   * @param test test definition
+   * @param response axios response
+   */
+  private verifyStatus(
+    test: TestDefinition,
+    response: AxiosResponse<any>
+  ): boolean {
+    if (test.response.status === response.status) {
+      this.logger.success("Status matched", { indent: 2 });
+      return true;
     } else {
-      TestLogger.error(
-        `State teardown request failed: ${
-          error.message
-        } (${TestTimer.formattedDiff(testTeardownStartTime)})`,
+      this.logger.error(
+        `Expected status ${test.response.status}, got ${response.status}`,
         { indent: 2 }
       );
+      return false;
     }
-    return false;
   }
-}
 
-/**
- * Check if an axios response status matches the expected status of a test.
- *
- * @param test test definition
- * @param response axios response
- */
-function verifyStatus(
-  test: TestDefinition,
-  response: AxiosResponse<any>
-): boolean {
-  if (test.response.status === response.status) {
-    TestLogger.success("Status matched", { indent: 2 });
-    return true;
-  } else {
-    TestLogger.error(
-      `Expected status ${test.response.status}, got ${response.status}`,
-      { indent: 2 }
-    );
-    return false;
-  }
-}
+  /**
+   * Check if an exios response body matches the expected body of an expected response definition.
+   *
+   * @param expectedResponse expected response
+   * @param response axios response
+   * @param typeStore reference type definitions
+   */
+  private verifyBody(
+    expectedResponse: DefaultResponseDefinition,
+    response: AxiosResponse<any>,
+    typeStore: TypeNode[]
+  ): boolean {
+    if (!expectedResponse.body) {
+      return true;
+    }
 
-/**
- * Check if an exios response body matches the expected body of an expected response definition.
- *
- * @param expectedResponse expected response
- * @param response axios response
- * @param typeStore reference type definitions
- */
-function verifyBody(
-  expectedResponse: DefaultResponseDefinition,
-  response: AxiosResponse<any>,
-  typeStore: TypeNode[]
-): boolean {
-  if (!expectedResponse.body) {
-    return true;
+    const jsv = new JsonSchemaValidator();
+    const schema = {
+      ...jsonTypeSchema(expectedResponse.body.type),
+      definitions: typeStore.reduce<{ [key: string]: JsonSchemaType }>(
+        (defAcc, typeNode) => {
+          return { [typeNode.name]: jsonTypeSchema(typeNode.type), ...defAcc };
+        },
+        {}
+      )
+    };
+    const validateFn = jsv.compile(schema);
+    const valid = validateFn(response.data);
+    if (valid) {
+      this.logger.success("Body compliant", { indent: 2 });
+      return true;
+    } else {
```

Comment:
You can omit the `else`. You'd have already returned. 
```
if (valid) {
  this.logger.success("Body compliant", { indent: 2 });
  return true;
}

this.logger.error(
  `Body is not compliant: ${jsv.errorsText(
      validateFn.errors
    )}\nReceived:\n${TestLogger.formatObject(response.data)}`,
    { indent: 2 }
  );
return false;
```
        { indent: 2 }
      );
      return false;